### ISSUE TITLE
Feature request: Allow multiple customizations of favico.ico, logo and URL for Postman doc publishing

### ISSUE THREAD

**Issue by gregarrayent** [a85/POSTMan-Chrome-Extension#3632]
## App Details:
```
Postman for Windows
Version 5.3.0
win32 10.0.15063 / x64

```

## Issue Report:
We are a SaaS company and we need to publish documentation for multiple tenants

Current Postman 
1.	Allows us to publish with a custom URL to match our tenant’s brand – That’s great and we can do this multiple times for multiple tenants
2.	Allows us to customize the logo – great – but only one logo? (we’d like as many logos as URLs)
3.	Does not allow us to customize the favico.ico and instead uses Postman’s – that is bad and we’d like to be able to customize it
See attached screen shot and comments
Bottom line
Each publication should allow us to select a URL, a logo and a favico.ico

Thanks
Greg

**Comment by prashantagarwal** [comment]
@gregarrayent Thanks for suggesting this feature.

**Comment by gregarrayent** [comment]
@prashantagarwal -- sure. And we really need it (we thought it was there when we decided to use Postman Docs -- my bad). When do you expect it will be implemented?

**Comment by sdnts** [comment]
@gregarrayent This hasn't been planned yet, so there's no ETA yet, but we'll keep you posted here :)

**Comment by godfrzero** [comment]
Logo and URL customization is already available. Favicon customization (a single favicon, for the team) is on the way.

**Comment by steffenschebesta** [comment]
@godfrzero any updates on the ETA for the favicon?

**Comment by godfrzero** [comment]
Update on the custom favicon, as posted on #2719:

> Public docs using a custom domain will now display a custom favicon if available. For example, if your custom doc is hosted at api.example.com, the favicon at https://example.com/favicon.ico will be used. Redirects are followed (unless the browser is configured otherwise). HTTPS is required, to avoid serving mixed content.
>
>The ability to upload a different favicon is still a work in progress, and I'll post an update here when it's out.

**Comment by pentatronix** [comment]
+1 for this.  We have several business units in our company and they all have separate logos associated with their API's.

**Comment by stonygrunow** [comment]
+1 for this. It's nuts our favicon and logo can't match our URL. We absolutely need this. Thanks!

**Comment by Raja-Simha** [comment]
@stonygrunow just to be clear, you can set up your own logo and favicon for public docs at $team.postman.com/settings/team/general. This logo and favicon will be used across all public docs. 

What you cannot do right now is set up a different logo and favicon for different public docs and that is what this feature request is about.

**Comment by eyalgrossdev** [comment]
Any updates on the _separate logo per collection / documentation_ feature? We would **really** like that as well. We're not holding our breathe though, there are a few other features that we and many others have requested which haven't had any traction for months. Disappointing.

**Comment by stonygrunow** [comment]
> @stonygrunow just to be clear, you can set up your own logo and favicon for public docs at $team.postman.com/settings/team/general. This logo and favicon will be used across all public docs.
> 
> What you cannot do right now is set up a different logo and favicon for different public docs and that is what this feature request is about.

Thanks Raja. Just to be clear, we have multiple instances of documentation, and yes, we would like the logo per-documentation to match the respective logo/favicon/all that stuff. We've shoe-horned in the best favicon/logo as possible for all of them, but that's a bit like having a single bicycle. Lots of people want a mountain bike and a road bike, and instead, we get 'bike' and have to hope it fits all situations.

**Comment by pentatronix** [comment]
To be clear, this is to allow MULTIPLE icons and logos to be used, for
instance multiple projects or in our case we provide API's for multiple
customers, each of whom needs their own logo. Our work around was to keep
getting free accounts and publishing each API to it's own account, but it
leads to a maintenance nightmare.

**Comment by michelleratcliffe** [comment]
Any chance this is still coming? I was just trying to implement it, and sad to find out that it cannot be done, yet 🤞
